R/cchart.R

```diff
@@ -286,11 +286,19 @@ CChart <- function(chart.type, x, small.multiples = FALSE,
     if (!is.null(attr(x, "signif.annotations")))
         signif.data.names <- unique(sapply(attr(x, "signif.annotations"), function(x) x$data))
 
-
     if (signif.show)
     {
+        colcmp.font <- list(
+            family = user.args$data.label.font.family,
+            color = user.args$data.label.font.color,
+            size = user.args$data.label.font.size * 1.3333)
```

Comment:
1.3333 would be good in a constant. I think it is also used elsewhere in this file.